code/game/objects/items/weapons/legcuffs.dm

```diff
@@ -60,12 +67,33 @@
 				to_chat(user, "<span class='danger'>You shouldn't be reading this message! Contact a coder or someone, something broke!</span>")
 				IED = null
 				return
+	if(istype(I, /obj/item/device/assembly/signaler))
+		if(IED)
```

Comment:
I wanted it to not be addable if there is something already attached to it, due to the sizes of the IED and the bear trap.